spec/index.md

```diff
@@ -651,6 +669,48 @@ The functionality exposed via the [The IC management canister](#ic-management-ca
 
 :::
 
+### Request: Sync Call {#http-sync-call}
+
+In order to make a synchronous update call to a canister, the user makes a POST request to `/api/v2/canister/<effective_canister_id>/sync_call`. The request body consists of an authentication envelope with a `content` map with the following fields:
+
+-   `request_type` (`text`): Always `sync_call`
+
+-   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
+
+-   `canister_id` (`blob`): The principal of the canister to call.
+
+-   `method_name` (`text`): Name of the canister method to call
+
+-   `arg` (`blob`): Argument to pass to the canister method
+
+The HTTP response to this request can have the following responses:
+
+-   202 HTTP status with empty body. Implying the request was accepted by the IC for further processing. Users should use [`read_state`](#http-read-state) to determine the status of the call.
```

Comment:
Nit: This may make more sense as the second entry, both because 202 comes after 200 and because it's the fallback behavior if a response is not produced in reasonable time.

Also, it may be worth explicitly stating that this is the fallback in case a response was not produced in reasonable time.